frontend/src/scenes/session-recordings/sessionRecordingLogic.test.ts

```diff
@@ -138,6 +142,144 @@ describe('sessionsPlayLogicV2', () => {
         })
     })
 
+    describe('loading session events', () => {
+        it('load events after metadata', async () => {
+            await expectLogic(logic, () => {
+                logic.actions.loadRecordingMeta('1')
+            })
+                .toDispatchActions(['loadRecordingMeta', 'loadRecordingMetaSuccess', 'loadEvents'])
+                .toMatchValues({
+                    eventsApiParams: {
+                        after: '2021-10-12T05:13:12+00:00',
+                        before: '2021-12-20T14:02:33+00:00',
```

Comment:
I think this should closely match the metadata in mocked response. A two month recording seems pretty wild!

Reply:
This should be fixed with using end_time instead of date math when calculating before and after bounds.